src/App.jsx

```diff
@@ -14,33 +16,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortOrder, setSortOrder] = useState(null);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortGoods = order => {
+    let sortedGoods;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    switch (order) {
+      case 'alphabet':
+        sortedGoods = [...goodsFromServer].sort();
```

Comment:
u can create variable for `[...goodsFromServer]`.
`const preparedGoods = [...goodsFromServer]`
```suggestion
        sortedGoods = preparedGoods.sort();
```